### DISCUSSION TITLE
Drop-in replacement for LS

### DISCUSSION THREAD

**Discussion by bisubus** [digitalfortress-tech/localstorage-slim#21] [Ideas]
The suggestion is to allow the library to be used as a drop-in replacement for LS. Currently method names are mismatched, some members don't exist, see https://developer.mozilla.org/en-US/docs/Web/API/Storage#properties

Aliases:

* `getItem()`
* `setItem()`
* `removeItem()`

Missing members:

* `valueOf()` (not documented but implemented on localStorage)
* `key()`
* `length`

**Comment by niketpathak** [comment]
Thank you for your valuable suggestion. Will surely consider it for the future release. 
Thanks again!
